SU-SWS/stanford_paragraph_types
# EARTH-000 Centered variant for callout blocks

# Not READY FOR REVIEW
# Summary
- Centered variant UI for callout blocks.

# Needed By (Date)
- EOS

# Urgency
- Low

# Steps to Test

1. Checkout branch
2. feature import all
3. create complex page with callout blocks and change the display variant.
4. verify it goes left and centered appropriately

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
